### PR TITLE
Added Issue-714 Split Array Largest Sum program in CPP

### PR DESCRIPTION
**Pull request by Atul-Kashyap**
Hey @rudrakshi99 , I added the Split Array Largest Sum c++ code in the "/Algo-Tree/C++" folder. I write code as per the guidelines.
Code passed all the test cases and if you find something wrong, please let me know.
Please merge it as soon as you can.
Thank You

issue link - #714

### CODE REVIEW

**Review by krishna-NIT — CHANGES_REQUESTED** [review]
Add this Que or Algo name at the end of    /Array/readme.md file

**Comment by Atul-Kashyap** [comment]
Hi @krishna-NIT,
Can you please merge this code.

**Comment by krishna-NIT** [comment]
resolve conflict

File: Code/C++/split_array_largest_sum.cpp [hunk 1]
```diff
@@ -0,0 +1,55 @@
+// Given an array nums which consists of non-negative integers and an integer m,
+// you can split the array into m non-empty continuous subarrays.
+
```
Comment (krishna-NIT) [inline Code/C++/split_array_largest_sum.cpp:3]: explain in brief

**Comment by krishna-NIT** [comment]
@Atul-Kashyap Why?

**Comment by Atul-Kashyap** [comment]
Hi @krishna-NIT ,
I create new branch and add the program,
Can you Please merge it ?
thank you

**Pull request closed without merging**